### PR TITLE
Append `Content-Length: 0` to request header when body of POST is empty

### LINKED ISSUE #35: leaveRoom() and leaveGroup() didn't work
got 411 bad request from server.
add request body can fix the problem.

[LINEBot.php:137](https://github.com/line/line-bot-sdk-php/blob/master/src/LINEBot.php#L137)
`return $this->httpClient->post($this->endpointBase . '/v2/bot/group/' . urlencode($groupId) . '/leave', ["dummy"=>"dummy"]);`

[LINEBot.php:148](https://github.com/line/line-bot-sdk-php/blob/master/src/LINEBot.php#L148)
`return $this->httpClient->post($this->endpointBase . '/v2/bot/room/' . urlencode($roomId) . '/leave', ["dummy"=>"dummy"]);`

### PR DESCRIPTION
**Pull request by moznion**
Fix #35

### CODE REVIEW

**Comment by moznion** [comment]
@Vaduz Review please

**Comment by Vaduz** [comment]
It would work but I'm wondering there is no test for this class.

File: tests/LINEBot/HTTPClient/CurlHTTPClientTest.php [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+                CurlHTTPClientTest::$reqMirrorPID = $out[0];
+                sleep(1); // XXX
+            }
+
+            return;
+        }
+
+        echo <<<"EOS"
+\n[WARN] req_mirror does not exist. Please try to execute `make install-devtool`
+[WARN] Skip CurlHTTPClientTest\n
```
Comment (Vaduz) [inline tests/LINEBot/HTTPClient/CurlHTTPClientTest.php]: I prefer not to use heredoc here, just use echo for each line instead.  Then we can use `PHP_EOL`.
``echo '[WARN] req_mirror does not exist. Please try to execute `make install-devtool`', PHP_EOL;``
`echo '[WARN] Skip CurlHTTPClientTest', PHP_EOL;`

And these messages should be inside else clause to describe function flow.

Comment (moznion) [inline tests/LINEBot/HTTPClient/CurlHTTPClientTest.php]: - PHP_EOL: Agree, I will change to use that.
- Else clause: I don't have much motivation to use `else`; I feel that's benefit is less or no.

Comment (Vaduz) [inline tests/LINEBot/HTTPClient/CurlHTTPClientTest.php]: This whole class tests should be failed if `req_mirror` is not available.

Comment (moznion) [inline tests/LINEBot/HTTPClient/CurlHTTPClientTest.php]: Does that mean `don't skip test, to fail instead`?
rel: https://github.com/line/line-bot-sdk-php/pull/38/files#diff-d0ccb18514d6f6cb88ff030b1b5697d0R69

Comment (Vaduz) [inline tests/LINEBot/HTTPClient/CurlHTTPClientTest.php]: Yes

Comment (moznion) [inline tests/LINEBot/HTTPClient/CurlHTTPClientTest.php]: Hmm, this is a pickle... But ok, I'll change it.

Comment (moznion) [inline tests/LINEBot/HTTPClient/CurlHTTPClientTest.php]: Done

**Pull request merged**